inputremapper/configs/migrations.py

```diff
@@ -258,24 +270,41 @@ def _otherwise_to_else():
                 symbol[0],
             )
 
-            preset_dict["mapping"][key] = symbol
+            preset_structure["mapping"][key] = symbol
 
         if not changed:
             continue
 
         with open(preset, "w") as file:
             logger.info('Changing otherwise to else for "%s"', preset)
-            json.dump(preset_dict, file, indent=4)
+            json.dump(preset_structure, file, indent=4)
             file.write("\n")
 
 
+def _input_combination_from_string(combination_string: str) -> InputCombination:
+    cfg = []
+    for event_str in combination_string.split("+"):
+        cfg.append(
+            {
+                k: int(v)
+                for k, v in zip(
+                    ("type", "code", "analog_threshold"), event_str.split(",")
+                )
+            }
+        )
+    return InputCombination(cfg)
```

Comment:
no abbreviated variable names please.

Would this be an easier to read equivalent?

```suggestion
    configs = []
    for event_str in combination_string.split("+"):
        type, code, analog_threshold = event_str.split(",")
        configs.append({
            type: type,
            code: code,
            analog_threshold: analog_threshold
        })
        
    return InputCombination(configs)
```

Reply:
the code in preset.py is not needed. It is a migration from beta to beta. which should be removed prior to the actual release.